spec/domain/queries/find_content_spec.rb

```diff
@@ -58,6 +58,112 @@
     )
   end
 
+  describe 'Other date ranges' do
```

Comment:
What about extracting the following in the describe: `Other date ranges`

```ruby
let(:this_month_date) { Date.today }
let(:edition1) { create :edition, base_path: '/path1', date: 2.months.ago, organisation_id: primary_org_id}
let(:edition2) { create :edition, base_path: '/path2', date: 2.months.ago, organisation_id: primary_org_id}
```

Shouldn't we also include here there 30 days?

Reply:
👍